src/settings.ts

```diff
@@ -44,6 +44,10 @@ document.querySelector('#content').innerHTML = `<h1>Gauntlet Settings</h1>
 <input type="checkbox" id="scroll-to-bottom">
 <input type="button" class="button" id="set-scroll-to-bottom" value="Set">
 </fieldset>
+<legend>Automatically Scroll To Bottom of Nation Pages</legend>
```

Comment:
Missing an opening tag for `<fieldset>`, causing some weird appearance for this setting on the settings page.
The `<legend>` element should be updated to be reflective of what this is doing since it appears to be a copy-paste of the previous section.
```suggestion
<fieldset>
<legend>Clear Dossier During Prep</legend>
```